app/forms/yeti_resource.rb

```diff
@@ -1,11 +1,8 @@
 # frozen_string_literal: true
 
-class YetiResource
+class YetiResource < ApplicationForm
```

Comment:
YetiResource and it's ancestors should not inherit from application form

it's not a form object